.gitignore

```diff
@@ -1,104 +1,136 @@
 # Logs
+
 logs
-*.log
-npm-debug.log*
+_.log
+npm-debug.log_
 yarn-debug.log*
 yarn-error.log*
-lerna-debug.log*
+lerna-debug.log\*
 
 # Diagnostic reports (https://nodejs.org/api/report.html)
-report.[0-9]*.[0-9]*.[0-9]*.[0-9]*.json
+
+report.[0-9]_.[0-9]_.[0-9]_.[0-9]_.json
 
 # Runtime data
+
 pids
-*.pid
-*.seed
-*.pid.lock
+_.pid
+_.seed
+\*.pid.lock
```

Comment:
why are these changed?

Reply:
Oh I was creating separate aws_config.json so I added it in gitignore but then figured I could just use .env to store aws credentials so I erased  it but I guess it somehow changed the file itself.

Reply:
pull this file from master

Reply:
`git checkout master .gitignore`

Reply:
okok